Apps/Examples/Examples/All Examples/OfflineManagerExample.swift

```diff
@@ -0,0 +1,422 @@
+                self?.state = .initial
+            }
+        }
+    }
+
+    private var state: State = .unknown {
+        didSet {
+            Log.warning(forMessage: "Changing state from \(oldValue) -> \(state)", category: "Debug")
+
+            switch (oldValue, state) {
+            case (_, .initial):
+                resetUI()
+                Log.warning(forMessage: "Enabling HTTP stack network connection", category: "Example")
+                NetworkConnectivity.getInstance().setMapboxStackConnectedForConnected(true)
+
+            case (.initial, .downloading):
+                // Can cancel
+                button.setTitle("Cancel Downloads", for: .normal)
+
+            case (.downloading, .downloaded):
+                Log.warning(forMessage: "Disabling HTTP stack network connection", category: "Example")
+                NetworkConnectivity.getInstance().setMapboxStackConnectedForConnected(false)
+                enableShowMapView()
+
+            case (.downloaded, .mapViewDisplayed):
+                showMapView()
+
+            case (.mapViewDisplayed, .finished),
+                 (.downloading, .finished):
+                button.setTitle("Reset", for: .normal)
+
+            default:
+                fatalError("Invalid transition from \(oldValue) to \(state)")
+            }
+        }
+    }
+
+    // MARK: - UI changes
+
+    private func resetUI() {
+        logger?.reset()
+        logView.textContainerInset.bottom = view.safeAreaInsets.bottom
+        logView.scrollIndicatorInsets.bottom = view.safeAreaInsets.bottom
+
+        progressContainer.isHidden = false
+        stylePackProgressView.progress = 0.0
+        tileRegionProgressView.progress = 0.0
+
+        button.setTitle("Start Downloads", for: .normal)
+
+        mapView?.removeFromSuperview()
+        mapView = nil
+    }
+
+    private func enableShowMapView() {
+        button.setTitle("Show Map", for: .normal)
+    }
+
+    private func showMapView() {
+        button.setTitle("Show Downloads", for: .normal)
+        progressContainer.isHidden = true
+
+        let mapView = MapView(frame: mapViewContainer.bounds, mapInitOptions: mapInitOptions, styleURI: .outdoors)
+        mapView.autoresizingMask = [.flexibleWidth, .flexibleHeight]
+        mapViewContainer.addSubview(mapView)
+
+        // Jump to center of our bounds
+        mapView.cameraManager.setCamera(to: CameraOptions(center: tokyoCoord,
+                                                          zoom: tokyoZoom))
+
+        // Add a point annotation that shows the point geometry that were passed
+        // to the tile region API.
+        mapView.on(.styleLoaded) { [weak self] _ in
+            guard
+                let annotationManager = self?.mapView?.annotationManager,
+                let coord = self?.tokyoCoord else {
+                return
+            }
+
+            annotationManager.addAnnotation(PointAnnotation(coordinate: coord))
```

Comment:
I think I will also add a point annotation on the map as well to be aligned

Reply:
what's the color for this point annotation?

Reply:
It's currently not configurable, so a default pin.

Reply:
I used a red dot in Android.